README.md

```diff
@@ -177,6 +178,31 @@ To run the tests on the master node, the pod needs to be scheduled on that node.
 
 The default labels applied to master nodes has changed since Kubernetes 1.11, so if you are using an older version you may need to modify the nodeSelector and tolerations to run the job on the master node.
 
+
+### Running in an AKS cluster
+
+1. Create an AKS cluster(e.g. 1.13.7) with RBAC enabled, otherwise there would be 4 failures
+
+1. Use the [kubectl-enter plugin] (https://github.com/kvaps/kubectl-enter) to shell into a node 
+`
+kubectl-enter {node-name}
+` 
+or ssh to one agent node
+could open nsg 22 port and assign a public ip for one agent node (only for testing purpose)
+
+1. config KUBECONFIG on the agent node:
```

Comment:
we don't need this step now, only need following steps:
```
kubectl-enter {node-name}
docker run --rm -v `pwd`:/host aquasec/kube-bench:latest install
./kube-bench node
```

Reply:
that is why i say OR ssh .. if somehow kubectl-enter tool breaks there is another way to get to the node..

Reply:
I mean step 3, it's not necessary now:
```
config KUBECONFIG on the agent node:

sudo chmod a+r /etc/kubernetes/certs/client.key
export KUBECONFIG=/var/lib/kubelet/kubeconfig
kubectl get no  # to check whether kubeconfig works
```

Reply:
ok.. removed now